storage/spanstore/utils.go

```diff
@@ -0,0 +1,17 @@
+package spanstore
+
+import (
+	"hash"
+	"math"
+)
+
+// HashBytes returns the uint64 hash value of bytes slice
+func HashBytes(h hash.Hash64, bytes []byte) uint64 {
```

Comment:
Let's not increase API scope, this function can be private until we need it to be public.

Reply:
this is because I'm using different package name for downsampling_writer_test.go and downsampling_writer.go. Let me know if that's necessary or not.